languages/elixir/exercises/concept/processes/.meta/design.md

```diff
@@ -0,0 +1,33 @@
+## Learning objectives
+
+- Know what a process is.
+- Know how to start a process.
+- Know how to send a message.
+- Know what a PID is.
+- Know about `self`.
+- Know about the `Process` module.
+
+## Out of scope
+
+- `GenServer`
+- `Agent`
+- links
+- monitors
+- supervisors
+- tasks
+
+## Concepts
+
+- `processes`
+- `pids`
+
+## Prerequisites
+
+- `atoms`
+- `recursion`
+- `pattern-matching`
+- `tuples`
+
```

Comment:
Not sure, but interaction with erlang's `:erlang.list_to_pid/1` was mentioned, should that be reflected here?

Reply:
I would rather remove the mention of `:erlang.list_to_pid/1` from `after.md` than make this exercise depend on Erlang. You definitely don't need it to solve it... but I think it's fine to just briefly mention a new topic in `after.md`, like I also did with `Agent` and `GenServer`. I'll ask an authority.